composer.json

```diff
@@ -36,7 +36,7 @@
     "require-dev": {
         "mikey179/vfsstream": "^1.6",
         "phpstan/phpstan": "^0.10.3",
-        "phpunit/phpunit": "^6.0"
+        "phpunit/phpunit": "^7.5.14"
```

Comment:
Hmm, can't we jump to 8.x directly? Is the test suite too antiquated for that?

Reply:
@Ocramius 

[`phpunit/phpunit:^8.0.0`](https://github.com/sebastianbergmann/phpunit/blob/8.0.0/composer.json#L24) requires PHP 7.2; this project still supports PHP 7.1.

Perhaps drop PHP 7.1 first?

Reply:
Updated!